frontend/src/router/router.jsx

```diff
@@ -25,13 +25,15 @@ import { Search } from '../pages/Search';
 import { NewTag } from '../pages/NewTag';
 import { Tags } from '../pages/Tags';
 import { Category } from '../pages/Category';
+import { EditDescriptionMetadataType } from '../pages/EditDescriptionMetadataType';
 
 const RouterComponent = () => { // eslint-disable-line arrow-body-style
   return (
     <Router history={history}>
       <Switch>
         <Route exact path="/" component={Splash} />
-        <Route path="/dataType/:name" component={MetadataByType} />
+        <PrivateRoute path="/dataType/description/:id" component={EditDescriptionMetadataType} />
```

Comment:
Should be municipality only. Submitting or voting for description as non-municipality user returns a 401 from the backend.